.github/workflows/main.yml

```diff
@@ -58,50 +58,3 @@ jobs:
     - name: Test with tox
```

Comment:
While you're at it, maybe change `3.11-dev` to `3.11` above (released now), and also re-check if pyside6 works on it nowadays?